date: 2020-12-14 21:37
not working yet
UnableToWorkException
Traceback (most recent call last):
  File "app.py", line 22, in <module>
    python_jun.work()
  File "/Users/antonhryb/Desktop/level/homework/24/models/employee.py", line 120, in work
    raise UnableToWorkException()
models.employee.UnableToWorkException: not working yet
